util/reggen/access.py

```diff
@@ -46,6 +47,18 @@ class HwAccess(JsonEnum):
     NONE = 4  # No access allowed
 
 
+SWACCESS_RDL_MAP = {
```

Comment:
Can you add a documentation comment above that explains that each row is a SystemRDL description of a set of access permissions? (It took me a couple of minutes to properly understand)